### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/filmstrip#1]
Some accessibility issues that should be addressed before this goes into production.
- replace the clickable links with buttons, so that they can be accessed with the keyboard.
- use an unordered list, rather than nested divs, for the filmstrip itself.
- add labels (maybe use `aria-label`) for the navigation buttons, so that they aren't read out as 'less than' and 'greater than'.

**Comment by amy-langley** [comment]
I have fixed these in the version of the filmstrip in pfe but need to come back and address these issues in the version of the control in this repository

**Comment by brian-c** [comment]
Why are there two versions?
